docs/developer-experience/artificial-intelligence.md

```diff
@@ -0,0 +1,63 @@
+# Artificial Intelligence
+
+There are a number of AI tools that can improve the developer experience. This article will discuss tooling that is available as well as advice on when it might be appropriate to use such tooling.
+
+## GitHub Copilot
+
+The current version of GitHub Copilot can provide code completion in Visual Studio Code. It is a VS Code extension that can be installed from the VS Code Marketplace. It is currently in beta and requires a GitHub account to use. It is free to use during the beta period. It is currently available for VS Code only, but there are plans to support other IDEs in the future. It is currently available for the following languages: JavaScript, TypeScript, Ruby, Go, Python, C#, C++, and Java. You can find more information about GitHub Copilot [here](https://copilot.github.com/).
+
+Developers on our team have found the following use-cases for GitHub Copilot:
```

Comment:
Can we reword this (and similar statements below). This wording fits great in blogs but feels a bit out of place in the playbook. Perhaps something like... "some example use-cases are"

Reply:
Agreed, the playbook should be more about hard cold facts.

Reply:
Yes, we can. However, realize that this content comes from the "sharing sprint" for a devcrew. We are not the product group saying facts about what use-cases are supported by GitHub Copilot. We are a devcrew who used the tools in an engagement - it can do a lot of things - these are the things that we as a crew found productive. Our guiding principal for sharing, in fact, has been that we don't share anything that we haven't actually vetted and proven to be useful on our team. If we just say "some example use-cases" then it seems like we are losing this important distinction.

Reply:
By sharing it in the EF playbook, the connotation is already that someone in CSE has found it useful, no need to call it out here.